tests/test_multisend.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from ape import networks
+
+from ape_safe import MultiSend
+from ape_safe.exceptions import SafeLogicError
+
+
+def test_asset(vault, token):
+    assert vault.asset() == token
+
+
+def test_default_operation(safe, token, vault):
+    with networks.ethereum.local.use_provider("foundry"):
```

Comment:
This won't work in CI unless you do the following:

1. Add both `ape-foundry` and `ape-alchemy` to `setup.py`.
2. add the environment variables needed for `ape-alchemy` to be able to work
3. Either configure via `ape-config.yaml` or call the `set_default_provider()` method in your tests so that mainnet's default provider (or foundry's mainnet upstream provider) is set to Alchemy.